CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 UNRELEASED
 ----------
-
+* Fix APIv3 single page endpoint for multiple translation versions
```

Comment:
```suggestion

* Fix APIv3 single page endpoint for multiple translation versions

```
A little caveat of our Changelog: In order to be correctly parsed by [awk magic](https://github.com/digitalfabrik/integreat-cms/blob/25b8706a1eb5004e96133fd73c567704238ba54e/.circleci/config.yml#L342), it needs exactly one new line after the heading and two new lines to the following release... Probably I should document this somewhere^^